lib/components/base/Pagination.vue

```diff
@@ -5,7 +5,7 @@
       :tabindex="page === 1 ? -1 : 0"
       class="left-arrow paginate has-icon"
       aria-label="Previous Page"
-      :href="linkFunction(page - 1)"
+      :href="linkFunction?.(page - 1)"
```

Comment:
```suggestion
      :href="linkFunction(page - 1)"
```

Optional chaining can be avoided by following the suggestion below.